src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -539,17 +539,17 @@ Status ServiceBasedNodeInfoAccessor::AsyncSubscribeToNodeChange(
   });
 }
 
-boost::optional<GcsNodeInfo> ServiceBasedNodeInfoAccessor::Get(
+absl::optional<GcsNodeInfo> ServiceBasedNodeInfoAccessor::Get(
```

Comment:
this is for mock testing. somehow boost::optional won't work due to print issues.